babel.config.js

```diff
@@ -1,5 +1,15 @@
 module.exports = {
-  presets: ['@babel/preset-typescript'],
+  presets: [
+    '@babel/preset-typescript',
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: 'current',
```

Comment:
```suggestion
          node: 8.3,
```
RN requires Node 8.3, we should be compatible